index.html

```diff
@@ -219,8 +219,21 @@
         </li>
       </ol>
       <div class="note">
-        The same-origin requirement prevents unauthorized nested third-party
-        content from requesting a wake lock on behalf of the top-level page.
+        <p>
+          The same-origin requirement prevents unauthorized nested third-party
+          content from requesting a wake lock on behalf of the top-level page.
+        </p>
+        <p>
```

Comment:
Do you mean create two separate notes? Both paragraphs are already wrapped in `<div class='note'>`